test/cli/basic/index.test.ts

```diff
@@ -0,0 +1,93 @@
+import fs from 'fs'
+import path from 'path'
+import { createTempDir, execBunchee, removeDirectory } from '../../utils/helper'
```

Comment:
I was thinking to get a helper that automatically create a test directory for each test, and then use the fixed input filename, in each folder.

So each test you only need to call the helper, and pass the directory (maybe just `__dirname`) and received the code, stderr, stdout etc. you just need to test them. Similar to what we had before

```js
// we have a generic helper `createTest`, that can define what does the executor do
const createCliTest = createTest({
   run(input, args) {
     // do cli work
     executeBunchee(...)
   }
})

// in each test, `createCliTest` can return the result
it(...) {
   const { code, stdout, distFiles, ... } = createCliTest({ directory: __dirname, args: [...] })
   // test stdout etc
}
```